Automattic/jetpack
# JS Components: Add RNA ConnectButton story

Simple PR that adds a story to the `<ConnectButton />` component.

* Depends on https://github.com/Automattic/jetpack/pull/23921
* Part of Part of https://github.com/Automattic/jetpack/issues/23708

#### Changes proposed in this Pull Request:

* JS Components: Add RNA ConnectButton story

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use Storybook
* Confirm you see the ConnectButton component there
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/77539/163192788-32c3d56d-ffec-4d50-8970-666cffa27d0d.png">
